.github/workflows/linux-builds.yml

```diff
@@ -0,0 +1,120 @@
+name: Build on Linux (Docker)
+
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        distro: # these names correspond to the files in .ci/$distro
+          - UbuntuFocal
+          - UbuntuBionic
+          - ArchLinux
+          - DebianBuster
+          - Fedora33
+        include:
+          - distro: UbuntuFocal
+            package: DEB
+            test: skip # UbuntuFocal has a broken qt for debug builds
+
+          - distro: UbuntuBionic
+            package: DEB
+
+          - distro: ArchLinux
+            package: skip # we are packaged in arch already
+            allow-failure: yes
+
+          - distro: DebianBuster
+            package: DEB
+
+          - distro: Fedora33
+            package: RPM
+            test: skip # Fedora is our slowest build
```

Comment:
We _might_ want to change this in the future, but for now I'm okay with skipping it